src/pudl/transform/params/ferc1.py

```diff
@@ -238,178 +244,219 @@
 FUEL_CATEGORIES: dict[str, set[str]] = {
```

Comment:
Note that these fuel category changes are mostly due to me putting them in alphabetical order. I know it's annoying to review, but it's way easier to add new ones / see if one is already there if it's in order.

I also made it so that the first fuel type displayed was the correct "category" so oil/gas is in the "oil" category. This moved some stuff out of the "other" category